service/k8sclient/k8sclient.go

```diff
@@ -522,13 +522,23 @@ func (c *K8sClient) DeleteXtraDBCluster(ctx context.Context, name string) error
 	return nil
 }
 
+// ErrNotFound should be returned when referenced resource does not exist
+// inside Kubernetes cluster.
+var ErrNotFound error = errors.New("resource was not found in Kubernetes cluster")
```

Comment:
let's move this to other errors on the top

Reply:
done